src/ops/OpsUtils.ts

```diff
@@ -0,0 +1,13 @@
+import {APIAuthor} from '../APIAuthor';
+
+export function newHttpHeader(au: APIAuthor): any {
+    var headers: any = {
+        'X-Kii-AppID': au.app.appID,
+        'X-Kii-AppKey': au.app.appKey
+    };
```

Comment:
We will not use `X-Kii-App` and `X-Kii-AppKey` anymore, since they are not necessary.